src/main/java/com/webcohesion/ofx4j/domain/data/tax1099/Tax1099R.java

```diff
@@ -214,18 +218,24 @@ public void setNetUnapEmp(String netUnapEmp) {
 	 * @return the distCode
 	 */
 	@Element ( name = "DISTCODE", required = true, order = 10 )
-	public String getDistCode() {
-		return distCode;
+	public List<String> getDistCodes() {
```

Comment:
![image](https://user-images.githubusercontent.com/12237604/72668309-702c2980-39f3-11ea-878f-814fbd2a31df.png)

Spec allows for multiple DISTCODEs